accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+| `data` | object | Allows manifests to include arbitrary data without risk of conflict. | No |
+
+## Workload Definitions
+
+Workload definitions take the following form:
+
+| Key | Type | Value | Required |
+|--|--|--|--|
+| `abstract` | bool | If `true`, this workload can only be extended, and is never exposed directly as an installable workload. Default is `false`. | No |
+| `kind` | string | Either `build` or `dev`. Default is `dev`. | No |
+| `description` | int | User-visible description for the workload. | Yes if dev and non-abstract |
+| `packs` | string array | IDs of the packs that are included in the workload. | No |
+| `extends` | string array | IDs of workloads whose packs should be included in this workload. | No |
+| `platforms` | string array | Limits the workload and workloads that extend it to only be shown and installed on these host platforms. The strings are RIDs. | False |
+
+At least one of `extends` or `packs` is required. If a workload resolves to zero packs, which is possible when some packs are platform-specific, it is implicitly abstract. A workload may transitively include the same pack multiple times or extend the same workload multiple times, and they will be deduplicated. As a consequence, recursive `extends` references are technically permitted but redundant and although they may result in validation warnings they will not result in runtime errors.
+
+The `kind` allows structuring workloads into smaller pieces so that their download and install footprint on CI is smaller. `build` workloads should contain only the packs that are used to build projects. They do not need descriptions as they are not expected to be shown in the UX - they will only be used via a CI-specific UX such as `dotnet workload restore --build-only`.
+
+Note that `extends` is functionally a dependency system and a way to factor out common sets of packages from workloads. By analogy to package managers such as apt-get and NuGet, workloads are metapackages that only permit unversioned dependencies and packs are packages that are only installable transitively.
+
+## Pack Definitions
+
+Pack definitions take the following form:
+
+| Key | Type | Value | Required |
+|--|--|--|--|
+| `kind` | string | Type of the pack. Valid values are `sdk`, `framework`, `library`, `template`, `tool`. | Yes |
+| `version` | string | Version of the pack. | Yes |
+| `alias-to` | object | Alias this definition to the provided package ID | No |
+
+The `alias-to` key allows aliasing this pack definition to a different underlying pack ID on each host platform. The keys are host platform RIDs, and the values are pack IDs. The `"*"` platform ID may be used to alias a pack on all platforms.
+
+For example, an SDK pack might contain a compiler that's a native executable and hence depends on the host platform:
+
+```json
+"foo.sdk.compiler": {
+    "version": "1.0",
+    "kind": "sdk",
+    "alias-to": {
+        "osx-x64": "foo.sdk.compiler.mac",
```

Comment:
Is it possible to have different version numbers for the mac and windows compiler packages?  What would that look like?